CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ Where possible, the source of algorithms should be referenced in the comments.
 
 Here are the current repositories for the Gonum project.
 If code you want to contribute doesn't quite fit in any of them, then please start a discussion on the [mailing list](https://groups.google.com/forum/#!forum/gonum-dev).
-Code can be found at [github.com/gonum/](https://github.com/gonum/)\<repo\>, and documentation at godoc.org/github.com/gonum/\<repo\>.
+Code can be found at [github.com/gonum/](https://github.com/gonum/)\<repo\>, and documentation at pkg.go.dev/github.com/gonum/\<repo\>.
```

Comment:
This is not right (it has been so for a long time). The doc links are not relative to github.com/gonum, so this should be `gonum.org/v1/\<repo\>`